packages/core/lib/commands/compile.js

```diff
@@ -58,8 +52,9 @@ const command = {
         internal: true,
         description:
           "Save the raw compiler results into <output-file>, overwriting any existing content."
-      }
-    ]
+      },
+    ],
+    allowedGlobalOptions: ["--network <name>", "--config <file>"]
```

Comment:
not necessary, but could we just make this:
```suggestion
    allowedGlobalOptions: ["network", "config"]
```

Reply:
Do we need to retain this info somehow though? It is pretty nice to have the descriptive `<name>` as a descriptor with each option. Any ideas on how to do this and keep that?

Reply:
Ah yeah, I wasn't clear in my comments - see how Faina has it now!